### ISSUE TITLE
`line_number:false` option does not remove line numbers from code block

### ISSUE THREAD

**Issue by erotavlas** [fluid-dev/hexo-theme-fluid#294]
I used the following option `line_number` in codeblock 

{% codeblock line_number:false %}
code
{% endcodeblock %}

however the line numbers remain.

I was referring to the documentation here  https://hexo.io/docs/tag-plugins.html

**Comment by zkqiang** [comment]
Sorry, `line_number` is not supported on highlight now.

It will be resolved in the future.

**Comment by erotavlas** [comment]
I'm not exactly sure where to find that, there is a package.json in the root of the blog folder but I don't know if contains what you are referring to

`{
  "name": "hexo-site",
  "version": "0.0.0",
  "private": true,
  "scripts": {
    "build": "hexo generate",
    "clean": "hexo clean",
    "deploy": "hexo deploy",
    "server": "hexo server"
  },
  "hexo": {
    "version": "4.2.1"
  },
  "dependencies": {
    "hexo": "^4.2.1",
    "hexo-browsersync": "^0.3.0",
    "hexo-generator-archive": "^1.0.0",
    "hexo-generator-category": "^1.0.0",
    "hexo-generator-index": "^1.0.0",
    "hexo-generator-tag": "^1.0.0",
    "hexo-renderer-ejs": "^1.0.0",
    "hexo-renderer-marked": "^2.0.0",
    "hexo-renderer-stylus": "^1.1.0",
    "hexo-server": "^1.0.0"
  }
}`

**Comment by zkqiang** [comment]
The simple solution: new a folder and `$hexo init` again

**Comment by erotavlas** [comment]
The package json looks identical when I `hexi init` a new blog, the only difference is version numbers

Is there a way to fix it in my current blog without creating an entirely new blog? My blog is on version 4.2 of hexo

**Comment by zkqiang** [comment]
Because I can't tell where the problem is, so creating new blog is an effective solution.
After creation, you only need to copy the configuration files and markdown files.

**Comment by erotavlas** [comment]
i think its because the older version that I'm using isn't using highlightjs, its using some other library that probably doesn't support turning off the line numbers.  https://jmblog.github.io/color-themes-for-google-code-prettify/ 

I upgraded and am in the process of transferring all my changes from the old configuration to the new one (it wasn't as easy as simply replacing it)  I was able to remove the line numbers but the style for the code block doesn't show up correctly.  I'll post a separate issue for it.
